qutip/distributions.py

```diff
@@ -125,11 +125,11 @@ def visualize_2d_colormap(self, fig=None, ax=None, figsize=(8, 6),
             fig, ax = plt.subplots(1, 1, figsize=figsize)
 
         if cmap is None:
-            cmap = mpl.cm.get_cmap('RdBu')
+            cmap = mpl.colormaps.get_cmap('RdBu')
```

Comment:
This is not supported by old matplotlib versions.
```suggestion
            cmap = mpl.colormaps['RdBu']
```
works from at least matplotlib 3.5 which is the oldest we test.